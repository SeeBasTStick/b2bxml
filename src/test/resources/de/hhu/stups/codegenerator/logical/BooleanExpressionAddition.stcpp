int main() {
    BooleanExpression logical;
    logical.simulate();
    cout << logical.getCounter();
    return 0;
}