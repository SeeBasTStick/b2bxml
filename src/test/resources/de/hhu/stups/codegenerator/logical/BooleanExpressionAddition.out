public static void main(String[] args) {
    BooleanExpression logical = new BooleanExpression();
    logical.simulate();
    System.out.println(logical.expr);
}